pkg/meta/tkv.go

```diff
@@ -1909,11 +1910,63 @@ func (m *kvMeta) CopyFileRange(ctx Context, fin Ino, offIn uint64, fout Ino, off
 	return errno(err)
 }
 
-/* TODO
-func (m *dbMeta) cleanupDeletedFiles() {}
+func (r *kvMeta) cleanupDeletedFiles() {
+	for {
+		time.Sleep(time.Minute)
+		klen := len(r.prefix) + 1 + 8 + 8
+		vals, _ := r.scanValues(r.fmtKey("D"), func(k, v []byte) bool {
+			return len(k) == klen && len(v) == 8
+		})
+		now := time.Now().Unix()
+		for k, val := range vals {
+			if r.parseInt64(val)+60 < now {
+				key := []byte(k)[len(r.prefix)+1:]
+				inode := r.decodeInode(key[:8])
+				length := uint64(r.parseInt64(key[8:]))
```

Comment:
use utils.FromBuffer might be better since we use that to encode; refer to: https://github.com/juicedata/juicefs/blob/main/pkg/meta/tkv.go#L2243-L2248

Reply:
Inode is encoded as little endian, so we can only use that for length.